src/js/index.js

```diff
@@ -44,61 +43,49 @@ function startFormService() {
 
 function startAuth() {
   import(/* webpackPrefetch: true, webpackChunkName: "auth" */ './auth')
-    .then(({ login, logout }) => {
-      login(startApp);
-      Radio.reply('auth', {
-        logout() {
-          logout();
-        },
-      });
+    .then(({ login }) => {
+      login(start);
     });
 }
 
+function startApps({ isForm, isOutreach }) {
+  if (isOutreach) {
+    startOutreach();
+    return;
+  }
+
+  if (isForm) {
+    startForm();
+    return;
+  }
+
+  startAuth();
+}
+
 document.addEventListener('DOMContentLoaded', () => {
   const rootRoute = getRootRoute();
-  const isForm = rootRoute === 'formapp';
-  const isOutreach = rootRoute === 'outreach';
   const isFormService = rootRoute === 'formservice';
 
   if (isFormService) {
     startFormService();
     return;
   }
 
+  const isForm = rootRoute === 'formapp';
+  const isOutreach = rootRoute === 'outreach';
+
   if ((_DEVELOP_ || _E2E_) && sessionStorage.getItem('cypress')) {
     versions.frontend = 'cypress';
+    appConfig.name = 'Cypress Clinic';
+    appConfig.cypress = sessionStorage.getItem('cypress');
```

Comment:
setting these things on the `appConfig` allowed less passing around of stuff and more standardization.